topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.md

```diff
@@ -122,8 +123,7 @@ The imzML composite file consists of two files. The first file contains the meta
 Before starting any analysis it is important to check the characteristics and quality of the MSI data. The MSI quality control tool creates a comprehensive PDF report that contains multiple descriptive visualizations of different data attributes. Here we will use the MSI quality control to get an idea about the m/z range and the shape of the chilli section. More details about the MSI quality control tool can be found in the tutorial [Mass spectrometry imaging 1: Loading and exploring MSI data]({{site.baseurl}}/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html).
 
 > ### {% icon hands_on %} Hands-on: Quality control 
->
-> 1. **MSI Qualitycontrol** {% icon tool %} with the following parameters:
+>1. {% tool [MSI_Qualitycontrol](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/cardinal_quality_report/cardinal_quality_report/2.6.0.1) %}
```

Comment:
@MarenStillger @foellmelanie the IDs needs to look like this:

`[MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy0)`

Otherwise, it does not work on other Galaxy instances.